### ISSUE TITLE
Standard install path conflict

### ISSUE THREAD

**Issue by torwag** [pycom/Pymakr#3]
Hi again,

the standard install paths of pymakr does conflict with package managers and other installations of eric6.
I guess it would been a better idea to isolate the pymakr install thus, people can run eric 6 and pymakr side by side.

**Comment by RalphHogenbirk** [comment]
I agree, we have this same issue open on the forum and have added this to our task list.
